src/browser/disabled_frame_group_tracker.ts

```diff
@@ -92,89 +101,110 @@ function handleApiMove(win: OpenFinWindow, delta: RectangleBase) {
             ? 2
             : 1
         : 0;
-    const moves = handleBoundsChanging(win, {}, newBounds.bounds, changeType);
-    const leader = moves.find(([w]) => w === win);
-    if (!leader || leader[1].moved(newBounds.bounds)) {
+    const moves = handleBoundsChanging(win, {}, applyOffset(newBounds, offset), changeType);
+    const {leader, otherWindows} = moves.reduce((accum: MoveAccumulator , move) => {
+        move[0] === win ? accum.leader = move : accum.otherWindows.push(move);
```

Comment:
:neckbeard: (total personal preference) I think an `if` is more readable here. Also perhaps name `[0]` for what it is (`OpenFinWindow`)